src/App.jsx

```diff
@@ -1,5 +1,6 @@
 import React from "react";
 import Router from "./router/Router";
+import "./App.css";
```

Comment:
💡  Maybe remove this.

Reply:
Addressed:  3bafec047159e4d0cbfb54a20908367e2ce5414d